File: app/src/main/java/fr/nspu/dev/recordvoicelearning/model/Folder.java
package fr.nspu.dev.recordvoicelearning.model;

import fr.nspu.dev.recordvoicelearning.utils.OrderPeerEnum;
import fr.nspu.dev.recordvoicelearning.utils.QuestionToAnswerEnum;

/**
 * Created by nspu on 18-02-03.
 */



public interface Folder extends CollectionObject{
    String getName();
    String getTypeQuestion();
    String getTypeAnswer();
    OrderPeerEnum getOrder();
    QuestionToAnswerEnum getQuestionToAnswer();
}

File: app/src/main/java/fr/nspu/dev/recordvoicelearning/model/Folder.kt
package fr.nspu.dev.recordvoicelearning.model

import fr.nspu.dev.recordvoicelearning.utils.OrderPeerEnum
import fr.nspu.dev.recordvoicelearning.utils.QuestionToAnswerEnum

/**
 * Created by nspu on 18-02-03.
 */


interface Folder : CollectionObject {
    val name: String?
    val typeQuestion: String?
    val typeAnswer: String?
    val order: OrderPeerEnum?
    val questionToAnswer: QuestionToAnswerEnum?
}
